packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added `ProfileInfo.profileManagerWillLoad` function to verify the credential manager will load before attempting to use it. [#2111](https://github.com/zowe/zowe-cli/issues/2111)
```

Comment:
Needs some clarification: What is "it" in this sentence? And what is the thing that is attempting to use "it"?

Reply:
I changed the wording, but in this case, we are just adding a function to verify the credential manager can load before calling another function that will attempt to use the credential manager. This is beneficial for things such as Zowe Explorer, which needs to know if there is going to be a problem loading credentials prior to actually attempting to access credentials from the credential vault.